src/site/content/en/blog/quictransport/index.md

```diff
@@ -0,0 +1,324 @@
+  console.error(`An error occurred: ${error}`);
+}
+```
+
+Similarly, use the <code>[abort()](https://developer.mozilla.org/en-US/docs/Web/API/WritableStreamDefaultWriter/abort)</code> method of the <code>[WritableStreamDefaultWriter](https://developer.mozilla.org/en-US/docs/Web/API/WritableStreamDefaultWriter)</code> to send a [QUIC RESET\_STREAM](https://tools.ietf.org/html/draft-ietf-quic-transport-27#section-19.4) to the server. When using <code>abort()</code>, the browser may discard any pending data that hasn't yet been sent.
+
+```js
+const ws = await transport.createSendStream();
+const writer = ws.getWriter();
+writer.write(...);
+writer.write(...);
+await writer.abort();
+// Not all the data may have been written.
+```
+
+#### ReceiveStream
+
+A <code>[ReceiveStream](https://wicg.github.io/web-transport/#receivestream)</code> is initiated by the server. Obtaining a <code>ReceiveStream</code> is a two-step process for a web client. First, it calls <code>receiveStreams()</code>, which returns a <code>ReadableStream</code>. Each chunk of that <code>ReadableStream</code>, is, in turn, a <code>ReceiveStream</code> that can be used to read <code>Uint8Array</code> instances sent by the server.
+
+```js
+async function readFrom(receiveStream) {
+  const reader = receiveStream.readable.getReader();
+  while (true) {
+    const {done, value} = await reader.read();
+    if (done) {
+      break;
+    }
+    // value is a Uint8Array
+    console.log(value);
+  }
+}
+
+const rs = transport.receiveStreams();
+const reader = rs.getReader();
+while (true) {
+  const {done, value} = await reader.read();
+  if (done) {
+    break;
+  }
+  // value is an instance of ReceiveStream
+  await readFrom(value);
+}
+```
+
+You can detect stream closure using the <code>[closed](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader/closed)</code> promise of the <code>[ReadableStreamDefaultReader](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader)</code>. When the QUIC stream is [closed with the FIN bit](https://tools.ietf.org/html/draft-ietf-quic-transport-27#section-19.8), the <code>closed</code> promise is fulfilled after all the data is read. When the QUIC stream is closed abruptly (for example, by <code>[STREAM_RESET](https://tools.ietf.org/html/draft-ietf-quic-transport-27#section-19.4)</code>), then the <code>closed</code> promise rejects.
+
+```js
+// Assume an active receiveStream
+const reader = receiveStream.readable.getReader();
+reader.closed.then(() => {
+  console.log('The receiveStream closed gracefully.');
+}).catch(() => {
+  console.error('The receiveStream closed abruptly.');
+});
+```
+
+#### BidirectionalStream
+
+A <code>[BidirectionalStream](https://wicg.github.io/web-transport/#bidirectional-stream)</code> might be created either by the server or the client.
+
+Web clients can create one using `createBidirectionalStream()`, which returns a promise for a `BidirectionalStream`.
```

Comment:
```suggestion
Web clients can create one using `QuicTransport.createBidirectionalStream()`, which returns a promise for a `BidirectionalStream`.
```

Reply:
I am not a fan of using the `InterfaceName.methodName()` syntax in examples when `methodName()` is an instance method, rather than a static method.

I think that the reference to `createBidirectionalStream()` is clear enough based on the example code, but if it needs additional clarity, I'd rather switch to the more precise "...the `createBidirectionalStream()` method of a `QuicTransport` instance..."

Reply:
I understand your reasoning. However there are several issues that override. It's been 80-some lines since the last reference to `QuicTransport`. I literally thought, 'what interface is that part of?' This isn't a working piece of code. `QuicTransport. createBidirectionalStream()` is really an identifier or key rather than a function call. Finally, identifying it this way would be more consistent with the way the site has been edited.

Your other option is to work `QuicTransport` into the text somehow to provide some kind of orientation.

Reply:
I'll go with "the `createBidirectionalStream()` method of a `QuicTransport` instance" in that case.

(If web.dev style is to use `ClassName.methodName()`, that might be an editorial decision to revisit. CC: @kaycebasques)

Reply:
I've added that language in a few places in https://github.com/GoogleChrome/web.dev/pull/3104/commits/4b489b8dc19ebb4039b329729ebeec0e3d99714b